net/newsboat/files/patch-Makefile.diff

```diff
@@ -6,21 +6,24 @@ The target "doc/example-config" is a dependency of "install-examples" only, so
 it runs in the destroot phase, add it to "all" to avoid that. 
 --- Makefile
 +++ Makefile
-@@ -18,7 +18,7 @@
+@@ -18,9 +18,9 @@
  DEFINES+=-DGIT_HASH=\"$(GIT_HASH)\"
  endif
  
 -WARNFLAGS=-Werror -Wall -Wextra -Wunreachable-code
 +WARNFLAGS=-Wall -Wextra -Wunreachable-code
  INCLUDES=-Iinclude -Istfl -Ifilter -I. -Irss
- BARE_CXXFLAGS=-std=c++11 -O2 -ggdb $(INCLUDES)
+-BARE_CXXFLAGS=-std=c++11 -O2 -ggdb $(INCLUDES)
++BARE_CXXFLAGS=-std=c++11 $(INCLUDES)
  LDFLAGS+=-L.
-@@ -81,7 +81,7 @@
+ 
+ PACKAGE=newsboat
+@@ -103,7 +103,7 @@
  TEXTCONV=./txt2h
  RM=rm -f
  
 -all: doc $(NEWSBOAT) $(PODBOAT) mo-files
 +all: doc $(NEWSBOAT) $(PODBOAT) mo-files doc/example-config
```

Comment:
Oh, I wish this was reported upstream. This makes total sense to me; I'll fix it in Newsboat shortly.

Reply:
Fixed by https://github.com/newsboat/newsboat/pull/927.

Reply:
Thank you.